src/fsharp/infos.fs

```diff
@@ -518,6 +518,25 @@ type OptionalArgInfo =
                 CallerSide (Constant v)
         else 
             NotOptional
+    //is there a better place for this? We need it both here and also in IlxGen.fs
+    static member FieldInitForDefaultParameterValueAttrib (Attrib (_,_,exprs,_,_,_,_)) =
+        let (AttribExpr (_,defaultValueExpr)) = List.head exprs
+        match defaultValueExpr with
+        | Expr.Const ((Const.String s),_,_) -> Some (ILFieldInit.String s)
```

Comment:
Yes, good pointer, will try to reuse...tomorrow evening, time for relaxing now. :)